mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -188,6 +188,12 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 		COMMENT {@return the center of this block position}
 		COMMENT
 		COMMENT @see Vec3d#ofCenter
+	METHOD method_49637 fromFloored (DDD)Lnet/minecraft/class_2338;
+		ARG 0 x
+		ARG 2 y
+		ARG 4 z
+	METHOD method_49638 fromPosition (Lnet/minecraft/class_2374;)Lnet/minecraft/class_2338;
```

Comment:
Most people would pass `Vec3d` here. Naming it consistent with `ofFloored` would be preferable.